lib/tmuxinator/project.rb

```diff
@@ -42,31 +42,47 @@ class Project
     attr_reader :custom_name
     attr_reader :no_pre_window
 
-    def self.load(path, options = {})
-      yaml = begin
-        args = options[:args] || []
-        @settings = parse_settings(args)
-        @args = args
-
-        content = render_template(path, binding)
-        YAML.safe_load(content, aliases: true)
-             rescue SyntaxError, StandardError => error
-               raise "Failed to parse config file: #{error.message}"
+    class << self
+      def load(path, options = {})
+        yaml = begin
+          args = options[:args] || []
+          @settings = parse_settings(args)
+          @args = args
+
+          content = render_template(path, binding)
+          YAML.safe_load(content, aliases: true)
+               rescue SyntaxError, StandardError => error
+                 raise "Failed to parse config file: #{error.message}"
+        end
+
+        new(yaml, options)
       end
 
-      new(yaml, options)
-    end
+      def parse_settings(args)
+        settings = args.select { |x| x.match(/.*=.*/) }
+        args.reject! { |x| x.match(/.*=.*/) }
 
-    def self.parse_settings(args)
-      settings = args.select { |x| x.match(/.*=.*/) }
-      args.reject! { |x| x.match(/.*=.*/) }
+        settings.map! do |setting|
+          parts = setting.split("=")
+          [parts[0], parts[1]]
+        end
 
       settings.map! do |setting|
         parts = setting.split("=", 2)
         [parts[0], parts[1]]
       end
```

Comment:
This looks like a bad rebase with master - we want to keep this version with the `split("=", 2)`, from https://github.com/tmuxinator/tmuxinator/pull/951